src/utils/country.ts

```diff
@@ -0,0 +1,29 @@
+import { BotContext } from '../context';
+import { getCountryCodeForText } from '../countries';
+
+export const validateCountry = (
+  countryName: string,
+  ctx: BotContext
+) => {
```

Comment:
this is quite a nice utility function, will be very handy to remove some duplication I see in the remote commands

Reply:
Glad to hear it !

Reply:
Refactored `registerRemoteMember.ts` to use this function.